### PR TITLE
Document selfie video widget

### LINKED ISSUE #982: Document selfie/new-front for video
It's here: https://github.com/opendatakit/collect/blob/HEAD/collect_app/src/main/java/org/odk/collect/android/widgets/VideoWidget.java#L106

### PR DESCRIPTION
**Pull request by grzesiek2010**
closes #982 

#### What is included in this PR?
It documents selfie video widget.

### CODE REVIEW

File: odk1-src/form-question-types.rst [hunk 1]
```diff
@@ -1691,6 +1699,29 @@ Records video, using the device camera.
   video, blinking, Please record a video of yourself blinking., Three times is probably sufficient.
 
 
+.. _selfie-video-widget:
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: Let's match the other selfie widget and call this `self-portrait-video-widget` and rename the other one `self-portrait-image-widget`

File: odk1-src/form-question-types.rst [hunk 2]
```diff
@@ -1691,6 +1699,29 @@ Records video, using the device camera.
...
+
+Selfie video widget
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: `Self portrait (*selfie*) widget`

File: odk1-src/form-question-types.rst [hunk 3]
```diff
@@ -1691,6 +1699,29 @@ Records video, using the device camera.
...
+
+.. image:: /img/form-widgets/selfie-video-widget-recorded.*
+  :alt: The Selfie Video form widget as displayed previously. The question text is "Selfie video widget". The hint text is "video type with new-front appearance". Below that are two buttons: Record Video and Play Video. Both buttons are enabled.
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name, label, appearance, hint
+
+  video, selfie_video, Selfie video widget, new-front, video type with new-front appearance
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: s/selfie_video/selfie-video/ (notice the hyphen to match the other questions)
s/Selfie video widget/`Self portrait (*selfie*) video widget`/

Then in the selfie image form, maybe we rename the widget to `Self portrait (*selfie*) image widget`

File: odk1-src/form-question-types.rst (hunk 1 above)
Comment (grzesiek2010) [reply odk1-src/form-question-types.rst]: Done.

File: odk1-src/form-question-types.rst (hunk 2 above)
Comment (grzesiek2010) [reply odk1-src/form-question-types.rst]: Done.

File: odk1-src/form-question-types.rst (hunk 3 above)
Comment (grzesiek2010) [reply odk1-src/form-question-types.rst]: Done.

**Pull request merged**